updates.jenkins.io.tf

```diff
@@ -1,15 +1,16 @@
 data "oci_core_images" "updates_jenkins_io" {
   compartment_id           = var.compartment_ocid
   operating_system         = "Canonical Ubuntu"
-  operating_system_version = "22.04"
+  operating_system_version = "20.04"
```

Comment:
Shouldn't this be part of a distinct PR?

Reply:
I don't think so, as the installation of this puppet version is not available in ubuntu 22.04, it make sense to have it in this PR.